blackbox/docs/configuration.txt

```diff
@@ -1888,6 +1888,52 @@ Javascript Language
   Setting to enable the Javascript language. As The Javascript language is an
   experimental feature and is not securely sandboxed its disabled by default.
 
+Authentication
+==============
+
+.. _host_based_auth:
+
+Host Based Authentication
+-------------------------
+
+.. NOTE::
+
+   Host Based Authentication is a CrateDB Enterprise Edition feature.
+   This Authentication only concerns connections over the ``PostgreSQL
+   Wire Protocol``.
+
+**auth.host_based**
+  | *Runtime:*     ``no``
+  | *Examples:*
+  |  - ``{user:crate, address: 172.16.0.0/16}``
+  |  - ``{method: trust}``
+  |  - ``{user:crate, address: 172.16.0.0/16, method:trust}``
+
+  The host based configuration is a list of predicates that users can specify
+  to restrict or allow access to CrateDB.
+
+  If ``auth.host_based`` is not set, the host based authentication is disabled.
+  If the setting ``auth.host_based`` is present and the configurations list
+  does not contain any entry, then no client can authenticate.
+
+  The meaning of the fields is as follows:
```

Comment:
in this context, it's okay for "meaning" to be singular. but because we're talking about multiple fields, the "is as" needs to be plural

consider a similar sentence: "the significance of the fields are as follows:"

I think this is ok. it certainly sounds right to my ear

if you wanted to make it singular, you would have to change it to "the meaning of each field is as follows" so that it is singular all the way through